app/workers/job_worker.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class JobWorker
```

Comment:
Can we call this maybe `ScheduledWorker` or something so it is consistent with the event receiver?